.jenkins/build_docker_images.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Set variable to allow parallel building in the Docker image creation
+export DOCKER_BUILDKIT=1
+
+# Get the dockerfiles for the images to be tested on Piz Daint
+dockerfiles=( $( ls docker/Dockerfile.* ) )
```

Comment:
Do we want these images to be built all at once in a single job, or one per job? If the compilation fails on one of the images, currently we would not be able to test the others.

Reply:
There's only 1 Docker image being tested currently so not an issue anymore